src/map/skill.cpp

```diff
@@ -1701,20 +1701,12 @@ int skill_additional_effect(struct block_list* src, struct block_list *bl, uint1
 	case AB_ADORAMUS:
 		sc_start(src,bl, SC_ADORAMUS, skill_lv * 4 + (sd ? sd->status.job_level : 50) / 2, skill_lv, skill_get_time2(skill_id, skill_lv));
 		break;
-	case WL_CRIMSONROCK:
-		sc_start(src,bl, SC_STUN, 40, skill_lv, skill_get_time(skill_id, skill_lv));
-		break;
 	case WL_COMET:
+		sc_start(src, bl, status_skill2sc(skill_id), 50, skill_lv, 10000); // !TODO: What's the rate?
```

Comment:
Sigma confirmed to me the rate is 100%
![image](https://user-images.githubusercontent.com/62680611/81745950-1062fe00-947c-11ea-9cee-32c4d83e322a.png)